SYCL/ESIMD/vec_arg_call_conv_ext.cpp

```diff
@@ -0,0 +1,87 @@
+//==-------------- vec_arg_call_conv_ext.cpp  - DPC++ ESIMD feature test ---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// Intel GPU is not really required, but the test does not make sense for
+// others.
+// REQUIRES: gpu
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl -Xclang -opaque-pointers -fsycl-device-only -Xclang -emit-llvm -o %t.comp.ll %s
```

Comment:
You used '-opaque-pointers' here and '-opaque-ptr' for the 1st test above. Do both of spellings supported?

Reply:
Thanks! No, just one, fixed.